Example/package.json

```diff
@@ -12,16 +12,16 @@
     "mobx": "^3.2.1",
     "mobx-react": "^4.2.1",
     "prop-types": "^15.5.10",
-    "react": "16.4.1",
-    "react-native": "0.56.0",
```

Comment:
Any reason to revert this here? Something that I might help? I was thinking about using the last version for everything. I've spent a couple of days in the past week just to make a few projects to work with these versions.

Reply:
Yes, please try to fix. I've got some babel-related error, connected with stage 0 or something like that.

Reply:
Cool - I think I know how to fix them. This might be related to the use of babel 7 (I had to upgrade a few other deps in order to make it to work - checking it).

Reply:
Got example working with latest `react` and `react-native`. 

![ezgif-5-da09ec92f0](https://user-images.githubusercontent.com/4285/43056377-645bd4ae-8e12-11e8-960b-29abba990285.gif)

Reply:
Also from:
```
✖ 538 problems (535 errors, 3 warnings)
```

To:
```
✖ 136 problems (133 errors, 3 warnings)
```